source/msam/test/test_layout.py

```diff
@@ -21,6 +21,10 @@ def setUp(self):
         This function is responsible for setting up the overall environment before each test
         """
         self.CLIENT_ERROR = ClientError({"Error": {"Code": "400", "Message": "SomeClientError"}}, "MockedFunction")
+    
+    def tearDown(self):
+        from chalicelib import layout
+        layout.DYNAMO_RESOURCE.reset_mock()
```

Comment:
need to reset for DYNAMO_CLIENT?

Reply:
no since DYNAMO_RESOURCE is the only mock in the test cases used for assertions.